src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -250,6 +250,102 @@ func (suite *DeltaTreeUnitSuite) TestFolderyMcFolderFace_AddTombstone() {
 	}
 }
 
+func (suite *DeltaTreeUnitSuite) TestFolderyMcFolderFace_SetPreviousPath() {
+	table := []struct {
+		name            string
+		id              string
+		loc             path.Elements
+		tree            *folderyMcFolderFace
+		expectErr       assert.ErrorAssertionFunc
+		expectLive      bool
+		expectTombstone bool
+	}{
+		{
+			name:            "no changes become a no-op",
+			id:              id(folder),
+			loc:             loc,
+			tree:            newFolderyMcFolderFace(nil, rootID),
+			expectErr:       assert.NoError,
+			expectLive:      false,
+			expectTombstone: false,
+		},
+		{
+			name:            "create tombstone after reset",
```

Comment:
could you please add one more test for a tree that has the node as live but has also been reset? I think this is a case worth calling out because incorrect implementation could lead to hard failures in kopia wrapper (i.e. if we accidentally generated collections for the live node and a tombstone but both had the previous path set we'd probably fail)

Given the code in delta_tree.go it's clear this will work, but this is more of a black box type test to ensure we don't accidentally make a future change that will lead to incorrect behavior

Reply:
sure can!